utils_cv/detection/data.py

```diff
@@ -132,3 +140,339 @@ def coco_labels() -> List[str]:
         "hair drier",
         "toothbrush",
     ]
+
+
+def extract_masks_from_labelbox_json(
+    labelbox_json: Union[str, Path],
+    old_dir: Union[str, Path],
+    new_dir: Union[str, Path] = None,
+) -> None:
+    """ Extract masks from Labelbox annotation JSON file.
```

Comment:
Can you provide a little more information what it does. Ie it reads in an annotation json file created by the LabelBox annotation UI, extracts the segmentation masks from all images, and then writes XXX to XXX. 
The other function also needs more info please.

Reply:
Added